ci-operator/step-registry/openshift-hyperfleet/e2e/test/openshift-hyperfleet-e2e-test-commands.sh

```
@@ -28,16 +28,14 @@ if [ -n "$E2E_REF" ]; then
-  cp -r /tmp/e2e-src/deploy-scripts /tmp/e2e/deploy-scripts
+  rm -rf /tmp/e2e/env /tmp/e2e/configs
+  cp -r /tmp/e2e-src/env /tmp/e2e/env
   cp -r /tmp/e2e-src/configs /tmp/e2e/configs
   cd -
   log "=== E2E build complete ==="
 fi
 
-cd "/tmp/e2e/deploy-scripts/"
-cp  .env.example .env
-source .env
+source /tmp/e2e/env/env.ci
```

Shellcheck (0.11.0)

[info] Not following: /tmp/e2e/env/env.ci was not specified as input (see shellcheck -x).

(SC1091)